freediscovery/data/ds_001/raw/0.7.6.28635.txt

```diff
@@ -1,4 +1,4 @@
-﻿From:           Eric Bass </o=teneo-test/ou=first administrative group/cn=recipients/cn=bass_eric>
+From:           Eric Bass </o=teneo-test/ou=first administrative group/cn=recipients/cn=bass_eric>
```

Comment:
ah, there was a strange first character there, like \u0eff or something. But you're right, we should use the original version of files, not just changing some encoding or cleaning special characters. And if we do, it has to be automated. I don't remember whether it was opening the file or creating dictionary that caused a problem. I'll restore this file and write a comment here